worlds/cccharles/Regions.py

```diff
@@ -0,0 +1,294 @@
+from BaseClasses import MultiWorld, Region, ItemClassification
+from . import CCCharlesItem
+from .Options import CCCharlesOptions
+from .Locations import (
+    CCCharlesLocation, loc_start_camp, loc_tony_tiddle_mission, loc_barn, loc_candice_mission, \
+    loc_tutorial_house, loc_swamp_edges, loc_swamp_mission, loc_junkyard_area, loc_south_house, \
+    loc_junkyard_shed, loc_military_base, loc_south_mine_outside, loc_south_mine_inside, \
+    loc_middle_station, loc_canyon, loc_watchtower, loc_boulder_field, loc_haunted_house, \
+    loc_santiago_house, loc_port, loc_trench_house, loc_doll_woods, loc_lost_stairs, loc_east_house, \
+    loc_rockets_testing_ground, loc_rockets_testing_bunker, loc_workshop, loc_east_tower, \
+    loc_lighthouse, loc_north_mine_outside, loc_north_mine_inside, loc_wood_bridge, loc_museum, \
+    loc_barbed_shelter, loc_west_beach, loc_church, loc_west_cottage, loc_caravan, loc_trailer_cabin, \
+    loc_towers, loc_north_beach, loc_mine_shaft, loc_mob_camp, loc_mob_camp_locked_room, \
+    loc_mine_elevator_exit, loc_mountain_ruin_outside, loc_mountain_ruin_inside, loc_prism_temple, \
+    loc_pickle_val, loc_shrine_near_temple, loc_morse_bunker
+)
+
+
+class CCCharlesRegion(Region):
+    regions = []
+
+def create_regions(world: MultiWorld, options: CCCharlesOptions, player: int):
```

Comment:
```suggestion
def create_regions(world: MultiWorld, options: CCCharlesOptions, player: int) -> None:
```
Other places explicitly annotate no return.